recipes/zmqpp/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans import AutoToolsBuildEnvironment
```

Comment:
Is there a reason you did not use the CMake support?

It would be much easier

Reply:
At first I had some issues with zmqpp cmake, so I switched to zmqpp Makefile. Now I'm working on CMake support, since Makefile support got quite complex.

Reply:
If you need help, push and add a comment, I'll do my best to get back to you =)